deno.lock

```diff
@@ -36,29 +36,13 @@
     "https://deno.land/std@0.186.0/path/posix.ts": "8b7c67ac338714b30c816079303d0285dd24af6b284f7ad63da5b27372a2c94d",
     "https://deno.land/std@0.186.0/path/separator.ts": "0fb679739d0d1d7bf45b68dacfb4ec7563597a902edbaf3c59b50d5bcadd93b1",
     "https://deno.land/std@0.186.0/path/win32.ts": "d186344e5583bcbf8b18af416d13d82b35a317116e6460a5a3953508c3de5bba",
```

Comment:
Came from build scripts and the https://github.com/denoland/deno_emit/blob/40a58651a1387d452325f6e9c9743a3f415e885b/js/_utils.ts#L5